src/main/java/io/github/christophermanahan/carnitas/Response.java

```diff
@@ -48,4 +52,17 @@ public boolean equals(Object object) {
           && headers.equals(response.headers)
           && body.equals(response.body);
     }
+
+    @Override
+    public String toString() {
+        return Stream.of(statusLine(), headers, List.of(body.orElse("")))
+          .flatMap(Collection::stream)
+          .filter(s -> !s.isEmpty())
+          .collect(Collectors.joining(Serializer.CRLF))
+          .concat(Serializer.CRLF);
```

Comment:
This introduces a circular dependency between the Response and the Serializer, yeah?